src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -231,6 +237,63 @@ def container_logs(cmd, resource_group_name, name, container_name=None, follow=F
             stream_args=(logs_client, resource_group_name, name, container_name, container_group.restart_policy))
 
 
+def container_exec(cmd, resource_group_name, name, container_name=None, exec_command=None, terminal_row_size=None, terminal_col_size=None):
+    """Start exec for a container. """
+
+    if platform.system() is WINDOWS_OS_NAME:
+        print("The container exec operation is currently not supported for Windows.")
```

Comment:
https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md
"Log to logger.error() or logger.warning() for user messages; do not use the print() function"